src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -148,7 +148,10 @@ private static IWebHost StartWith(string url, Action<IServiceCollection> configu
         public static IWebHostBuilder CreateDefaultBuilder(string[] args)
         {
             var builder = new WebHostBuilder()
-                .UseKestrel()
+                .UseKestrel((builderContext, options) =>
```

Comment:
Update the remarks above (and related ones on other overloads) to mention that Kestrel will be configured, e.g.:
`///     use Kestrel as the web server and configure it using the application's configuration providers,`

Reply:
Updated